composer.json

```diff
@@ -22,6 +22,7 @@
     ],
     "require": {
         "php": "^7.2",
+        "ext-json": "*",
```

Comment:
Why do we need this?

Reply:
Because of usage of `json_encode` and `json_decode` methods